app/routes/table.$.tsx

```diff
@@ -43,15 +43,9 @@ function handleRedirects(context: LoaderFunctionArgs) {
     {from: '',to: '',},
   ]
 
-  reactTablev7List.forEach((item) => {
+  reactTableV7List.forEach((item) => {
     if (url.pathname.startsWith(`/table/v7/${item.from}`)) {
-      throw redirect(
-        `/table/v8/${
-          item.to
-        }?from=reactTableV7&original=https://github.com/TanStack/table/tree/v7/docs/src/pages/${
-          item.from
-        }${item.from ? '.md' : ''}`
-      )
+      throw redirect(`/table/v8/${item.to}?from=reactTableV7`)
```

Comment:
It seems like this redirect will never, because routes like `/table/v7/docs/api/useColumnOrder` are handled in `app/routes/table.$version.docs.$.tsx` and not this file. 

This is what happens to routes like `/table/v7/docs/api/useColumnOrder`:
```sh
/table/v7/docs/api/useColumnOrder
/table/latest/docs/api/useColumnOrder #1st redirect
/table/latest/docs/introduction #2nd redirect
```

Reply:
Yeah that's what I thought, I originally removed these because I didn't think they did anything. What do you think about removing? I guess they could be put in the `docs.$` level?

Reply:
Yeah, I think they should be there. Btw the routes are messed up anyways. See this one:
```tsx
{from: 'docs/api/useColumnOrder',to: 'docs/api/features/column-ordering',}
```
The value for the `to` property should be `docs/guide/column-ordering`. 🤣